scpsl/exiled/egg-s-c-p--s-l--exiled.json

```diff
@@ -1,13 +1,12 @@
 {
-    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
```

Comment:
no, now you overwrote the pelican egg with the pterodactyl one, revert and do it to the right file, if you don't have a pelican instance, that is not an issue then only do the pelican one